### PR TITLE
Feature-IOPS-2736-Primary-Sample-State

### PR DESCRIPTION
**Pull request by KazeemHamzat**
Feature-IOPS-2736-Primary-Sample-State

### CODE REVIEW

**Review by omarisgreat — COMMENTED** [review]
This may need holding off until we have a final valueset from adam (the sample type/state) discussions

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:23]: This may be under Specimen.container.type depending on the type of information the MDS is specifying, needs clarification from Adam/the genomics SMEs

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:36]: May need a more specific code to differentiate this from a normal BCT. Needs terminology input

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:47]: Need to clarify whether we are specifying the tube type or the specimen conditions, e.g. tube with EDTA or specimen with EDTA

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:57]: May need to develop CodeSystem to capture codes not in SNOMED CT

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:66]: Would this be more like: 441652008 | Formalin-fixed paraffin-embedded tissue specimen (specimen) |

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:76]: Might need a 261013005 | Fresh (qualifier value) | on a specimen type (ideally all would be qualifiers as they would qualify the state of a specimen

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:87]: 441479001 | Fresh tissue specimen (specimen) | seems more appropriate, though if we are coding container types, the concept type does not match

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:98]: Could also be 838941000000105 | Plasma specimen with lithium heparin (specimen) | if we are qualifying the specimen itself

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:109]: Might need higher level concept such as specimen state or specimen condition (neither in SNOMED CT)

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:121]: Need to confirm this will always be within an evacuated BCT

File: ConceptMap/ConceptMap-Genomics-primary-sample-state.json
Comment (omarisgreat) [inline ConceptMap/ConceptMap-Genomics-primary-sample-state.json:132]: Streck is a company which provides multiple tube types, need to confirm what exactly is being referenced here

**Review by omarisgreat — CHANGES_REQUESTED** [review]
Many inconsistencies in the MDS dataset, these need to be reviewed by Adam and the genomic SMEs to concretely defined the concepts

**Comment by KazeemHamzat** [comment]
May delay approval until the associated targetUri values are available in UKCore.

**Review by omarisgreat — APPROVED** [review]
Happy for you to merge in once the UK Core set is approved

**Pull request merged**
